### ISSUE TITLE
django-hamlpy==1.4.2 not compatible with Django==3.1.4

### ISSUE THREAD

**Issue by cscheng** [nyaruka/django-hamlpy#89]
I'm using pip version 20.3.3 (Python 3.7 btw):

```
$ pip install django-hamlpy==1.4.2 Django==3.1.4
...
ERROR: Cannot install Django==3.1.4 and django-hamlpy==1.4.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested Django==3.1.4
    django-hamlpy 1.4.2 depends on django<3.0 and >=2.1
```

It seems like this issue isn't caught by GitHub CI, due to poetry wrapping the `pip install`. If you look at a [recent CI job](https://github.com/nyaruka/django-hamlpy/runs/1579017696?check_suite_focus=true), specifically for Django 3.1.4, and expand the "Initialize environment" step, you'll see the same pip error.

**Comment by rowanseymour** [comment]
Ah right I think latest pip just go stricter about conflicts. Will fix

**Comment by rowanseymour** [comment]
This should be fixed now in latest 1.4.3

**Comment by cscheng** [comment]
Thanks for the quick response and fix!
